.github/windowdesc-ini_key.py

```diff
@@ -0,0 +1,48 @@
+"""
+Script to scan the OpenTTD source-tree for ini_key issues in WindowDesc entries.
+"""
+
+import glob
+import os
+import re
+import sys
+
+def scan_source_files(path, errors, ini_keys):
```

Comment:
What you mostly see here, is that `ini_keys=None`, and the recursive call gives `ini_keys`. And then a line in this function:

```
if ini_keys is None:
  ini_keys = set()
```

This way, the outside doesn't know the inside of this function.